modules/portworx/examples/simple/outputs

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Please delete the generated `outptus.tf` file and rename this one with the .tf extension